frontend/lib/widgets/register_button.dart

```diff
@@ -0,0 +1,20 @@
+
+import 'package:flutter/material.dart';
+
+class RegisterButton extends StatelessWidget {
+  const RegisterButton({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return ElevatedButton(
+      onPressed: null, 
+       //style: ButtonStyle(padding: EdgeInsets.all(0.0), ),
+      child: const Text('Register', style: TextStyle(fontSize: 11)) );
+  }
+
+  // void _registerPressed() {
+    //   Navigator.push(context, MaterialPageRoute(builder: (context) => BrowseEvents()));
+    // }
+}
```

Comment:
No commented code

Reply:
Removed this in commit 029e4c6f432a732625dbf521275750643f5e2128